.env.test.example

```diff
@@ -12,5 +12,7 @@ TREASURY=
 FX_ROOT=
 FX_CHILD=
 CHECKPOINT_MANAGER=
-DEFENDER_TEAM_API_KEY=
-DEFENDER_TEAM_API_SECRET_KEY=
+DEFENDER_TEAM_API_KEY=1234567890ABCDEF1234567890abcdef
+DEFENDER_TEAM_API_SECRET_KEY=1234567890ABCDEF1234567890abcde1234567890ABCDEF1234567890abcdeff
+FORKING_ROOT_BLOCK_NUMBER=20700204
```

Comment:
Are we using any fork test ? If yes, Can we make this dynamic sth like setForkBlock() function in helpers so that tests can run on dynamic blocks.

Reply:
Yes, we use mainnet forking for the tests. The reason why we have a fixed block number here is caching. The first test run is fairly slow, but the subsequent test runs will be blazingly fast, as all of them will hit a local cache for contract data rather than querying mainnet RPC nodes.